src/api/server.rkt

```diff
@@ -228,6 +240,17 @@
         (hash-remove! waiting job-id)]
        ; Get the result for the given id, return false if no work found.
        [(list 'result job-id handler) (place-channel-put handler (hash-ref completed-work job-id #f))]
+       [(list 'timeline job-id handler)
+        (define wid (hash-ref current-jobs job-id #f))
+        (when (not (false? wid))
```

Comment:
`when wid` I think...

Reply:
Actually since you have a `when` and then a `when not` you just want a `cond` statement here.